backend/src/main/java/gov/cdc/usds/simplereport/db/model/EternalAuditedEntity.java

```diff
@@ -0,0 +1,33 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * Intermediate base class for entities that should be tagged as "deleted" in
+ * the database rather than actually being deleted.
+ */
+@MappedSuperclass
+public abstract class EternalAuditedEntity extends AuditedEntity
```

Comment:
yes, it was intended as a rename, plus implementing the `Eternal` interface. i will remove `EternalEntity`.

i disagree, i think there's a good case for renaming it now. we've now discovered a use case where a table (`api_user`) is eternal but not audited, which makes it dangerous having `EternalEntity` be secretly `Audited` as well, since a developer can unknowingly pull in a DB model type with the wrong attributes.

Reply:
This is not the proper venue for continuing this discussion, so I am going to punt it to slightly more synchronous forms of conversation. 🙂